Framework/PythonInterface/mantid/simpleapi.py

```diff
@@ -253,8 +253,14 @@ def StartLiveData(*args, **kwargs):
     except KeyError:
         pass
 
-    # Handle unpacking / workspace names from lhs
+    # LHS Handling currently unsupported for StartLiveData
     lhs = _kernel.funcinspect.lhs_info()
+    if lhs[0] > 0:  # Number of terms on the lhs
+        raise RuntimeError("Assigning the output of StartLiveData is currently "
```

Comment:
OK is this the fix until we figure out how to deal with optional output workspaces?